modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/sidebar/Sidebar.es.js

```diff
@@ -26,7 +26,6 @@ export default () => {
 		field,
 		formReportRecordsFieldValuesURL,
 		isOpen,
-		language,
```

Comment:
@alinedoleron, I didn't understand this commit. Can you please clarify? The commit message is inconsistent with the code.

Reply:
Before Bui's review I was getting the language that came from back end and I was sending through props. In this commit I was removing the prop of language and getting the language of themeDisplay.

Reply:
But it's weird. Is it possible this commit had had more than this line but those line are in common with the code on master? Beacuse it's related to this line of current Sidebar.es.js file: `<List data={data} type={type}></List>`

Reply:
Maybe. I'm not sure.